### PR TITLE
Added support for stripe options during initialization (stripeAccount, apiVersion, locale)

### PR DESCRIPTION
**Pull request by Bibendus83**
I added the new parameter `stripeOptions` that allows to pass `stripeAccount`, `apiVersion` and `locale` parameters to Stripe during initialization.
Parameter `stripeAccount` is required when direct charging Stripe Connect accounts (https://stripe.com/docs/connect/direct-charges#create-a-charge).
Check stripe documentation here: https://stripe.com/docs/js/initializing#init_stripe_js-options

Considering this repository seems quite dead and hardly the pull request will be accepted, if you want Stripe Connect support you can add directly my fork to your project with:
`yarn add https://github.com/Bibendus83/vue-stripe-elements/#feature/add_support_for_stripe_options`
or
`npn add https://github.com/Bibendus83/vue-stripe-elements/#feature/add_support_for_stripe_options`

### CODE REVIEW

**Comment by Bibendus83** [comment]
Adding reference to issues #106 and #116 for Stripe Connect support.

File: src/stripeElements.js [hunk 1]
```diff
@@ -31,26 +32,27 @@ export const baseStyle = {
   }
 }
 
-function init(key, options = {}) {
+function init(key, stripeOptions = {}, elementOptions = {}) {
   if (typeof key === "object" && typeof key.elements === "function") {
     Stripe.instance = key
   }
 
+  console.log(stripeOptions);
```
Comment (tectiv3) [inline src/stripeElements.js:40]: forgotten console log:)

Comment (Bibendus83) [reply src/stripeElements.js]: fixed :P

**Comment by yurikoval** [comment]
@sedrickcz could you take a look at this?

**Comment by softbeehive** [comment]
Apologies for a long long wait. Thanks for raising the issue, fixed in #120

**Pull request closed without merging**